_posts/2018-07-17-its-a-lot-more-snappy.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title: "\"It's A Lot More Snappy\""
+date: 2018-07-17
+categories: PWA SPA JavaScript
+author: Rick Foxcroft
+image: img/offline_logo.png
+---
+
+## _‘A real world application of PWA technology’_
+
+Recently Mechanical Rock undertook the journey of building a progressive web app (PWA) for a client here in Perth. A PWA uses modern web capabilities to deliver a native app-like experience by providing offline support and resource caching. The application we were building needed to provide users the ability to raise issues on equipment in geographically isolated areas. Due to this isolation and unreliable networks, it was important that the app could still function while offline and store data, locally on devices. We decided to build a PWA so that previously fetched data could be stored locally on the device and the application could be used without a network connection and still serve the data to the user.
```

Comment:
Excellent!

> Due to this isolation and unreliable networks, it was important that the app could still function while offline and store data, locally on devices. We decided to build a PWA so that previously fetched data could be stored locally on the device and the application could be used without a network connection and still serve the data to the user.

Repetition here - can it be pruned?